src/_assets/css/_showcase.scss

```diff
@@ -1,6 +1,6 @@
 .showcase {
   background-color: $site-color-codeblock-bg;
-  font-family: $site-font-family-alt;
+  font-family: "Google Sans", "Roboto", sans-serif;
```

Comment:
Assuming there is no preexisting variable for this?

Reply:
Yes, that variable exists, thank you.